cmd/tailscale/cli/status.go

```diff
@@ -221,9 +222,43 @@ func runStatus(ctx context.Context, args []string) error {
 		outln()
 		printHealth()
 	}
+	printFunnelStatus(ctx)
 	return nil
 }
 
+// printFunnelStatus prints the status of the funnel, if it's running.
+// Prints nothing if the funnel is not running.
```

Comment:
// It prints nothing ...

(complete sentence)

Reply:
Done.